src/views/Home/PreviousHackathons/index.view.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react"
+/*import SpeakerCard, {
```

Comment:
It would be great to remove these unnecessary comments left over from the refactor.